src/main/resolvers/consul.ts

```diff
@@ -193,53 +193,77 @@ export function consulResolver(): IRemoteResolver {
             )
         },
 
-        async get<T = any>(key: string): Promise<T> {
```

Comment:
I don't know that we need to update the existing resolvers to use the altKey. I think they can ignore it and new resolvers that want it can use it. Any current use case for the existing resolvers to use it?

Reply:
For testing I'd just create a dummy resolver that handles the new `altKey`.